app/src/features/projects/CreateProjectPage.tsx

```diff
@@ -159,6 +163,38 @@ const CreateProjectPage: React.FC = () => {
   const [openDraftDialog, setOpenDraftDialog] = useState(false);
 
   const [draft, setDraft] = useState({ id: 0, date: '' });
+  const [initialProjectFieldData, setInitialProjectFieldData] = useState<ICreateProjectRequest>({
+    coordinator: ProjectCoordinatorInitialValues,
+    permit: ProjectPermitFormInitialValues,
+    project: ProjectDetailsFormInitialValues,
+    objectives: ProjectObjectivesFormInitialValues,
+    species: ProjectSpeciesFormInitialValues,
+    location: ProjectLocationFormInitialValues,
+    iucn: ProjectIUCNFormInitialValues,
+    funding: ProjectFundingFormInitialValues,
+    partnerships: ProjectPartnershipsFormInitialValues
+  });
+
+  // Get draft project fields if draft id exists
+  useEffect(() => {
+    const getDraftProjectFields = async () => {
+      const response = await biohubApi.draft.getDraft(queryParams.draftId);
+
+      setHasLoadedDraftData(true);
+
+      if (!response || !response.data) {
+        return;
+      }
+
+      setInitialProjectFieldData(response.data);
+    };
+
+    if (!queryParams.draftId) {
+      return;
+    }
+
+    getDraftProjectFields();
```

Comment:
Small optimization: can we check for the existence of queryParams.draftId before we make this call? If its null then we can skip this altogether and it will save a bit of loading time.

Reply:
We also probably want to check for hasLoadedDraftData === false too, so that we only ever load it once

Reply:
Wait, ignore my first comment, I cant read apparently.  Second one is still valid I think...?

Reply:
I already am making a check and this line will not be executed if draftId does not exist 😄

Reply:
hasloadeddraftdata is based on queryparams.draftId so we can just check it alone